### PR TITLE
[Frontend] Events carousel on Home page

### PR DESCRIPTION
**Pull request by CamronLDNF**
## Description
Adds an events carousel to the home page.

## PivotalTracker
[Create Events carousel on Home page](https://www.pivotaltracker.com/story/show/163635794)

## Tasks
- Create EventsCarousel component
- Write acceptance and component tests

## Modules added
- moment
- nuka-carousel

## Screenshots 
image

### CODE REVIEW

File: client/we_meet_client/src/components/ui-components/EventsCarousel.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    let events = this.state.events;
+    let eventsList = events.map(event => {
+      return (
+        <Box key={event.id} inlineBlock >
+          <Card
+            className="card"
+            border
+            shadow
+            maxW="sm"
+            style={{
```
Comment (oliverochman) [inline client/we_meet_client/src/components/ui-components/EventsCarousel.js]: can you extract the styling to a separate file?

File: client/we_meet_client/src/__features__/homepage_navbar.feature.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ describe('Homepage', () => {
         await page.reload();
     });
 
-    describe('navbar', () => {
+    describe('displays correct buttons in navbar', () => {
```
Comment (Jon1969Edwards) [inline client/we_meet_client/src/__features__/homepage_navbar.feature.js:12]: Can this be worded differently?  How would one display an incorrect button?

File: client/we_meet_client/src/components/ui-components/EventsCarousel.js (hunk 1 above)
Comment (tochman) [reply client/we_meet_client/src/components/ui-components/EventsCarousel.js]: Done

File: client/we_meet_client/src/__features__/homepage_navbar.feature.js (hunk 2 above)
Comment (tochman) [reply client/we_meet_client/src/__features__/homepage_navbar.feature.js:12]: We just want to make sure the buttons are displayed.

**Pull request merged**